{
  "uid" : "9b4039697c534759",
  "name" : "T33: Update an existing task with an invalid priority.",
  "fullName" : "tests.test_task_management.TestTaskManagement#test_update_task_with_invalid_priority",
  "historyId" : "fafa0064ecdb363ac8de4c6fb4cb98bc",
  "time" : {
    "start" : 1738503493400,
    "stop" : 1738503495010,
    "duration" : 1610
  },
  "status" : "failed",
  "statusMessage" : "AssertionError: Expected status code 400, but got 200. Response: {'status': 'success', 'task': {'_id': '679f75466eb12444f2b46d5e', 'title': 'Test thank subject.', 'start': '07:41:52', 'end': '21:50:11', 'priority': 'someone', 'category': 'in-progress', 'owner': '679f75456eb12444f2b46d5a', 'date': '2025-01-27T00:00:00.000Z', 'createdAt': '2025-02-02T13:38:14.425Z', 'updatedAt': '2025-02-02T13:38:14.910Z', '__v': 0}}",
  "statusTrace" : "self = <test_task_management.TestTaskManagement object at 0x7fcf9fd10f90>\n\n    @pytest.mark.order(33)\n    @allure.title(\"T33: Update an existing task with an invalid priority.\")\n    def test_update_task_with_invalid_priority(self):\n        user = self.api_user_auth.create_user()\n        task = self.api_task.create_task(Token=user.data.accessToken)\n>       self.api_task.update_task_with_invalid_priority(Token=user.data.accessToken, task_id=task.id)\n\ntests/test_task_management.py:83: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \nservices/Task/api_task.py:160: in update_task_with_invalid_priority\n    self.assert_status(response, 400)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <services.Task.api_task.TaskAPI object at 0x7fcf9e41ea50>\nresponse = <Response [200]>, expected_status = 400\n\n    def assert_status(self, response, expected_status):\n>       assert response.status_code == expected_status, \\\n            (f\"Expected status code {expected_status}, but got {response.status_code}. Response: {response.json()}\")\nE       AssertionError: Expected status code 400, but got 200. Response: {'status': 'success', 'task': {'_id': '679f75466eb12444f2b46d5e', 'title': 'Test thank subject.', 'start': '07:41:52', 'end': '21:50:11', 'priority': 'someone', 'category': 'in-progress', 'owner': '679f75456eb12444f2b46d5a', 'date': '2025-01-27T00:00:00.000Z', 'createdAt': '2025-02-02T13:38:14.425Z', 'updatedAt': '2025-02-02T13:38:14.910Z', '__v': 0}}\n\nutils/helper.py:13: AssertionError",
  "flaky" : false,
  "newFailed" : false,
  "beforeStages" : [ {
    "name" : "_xunit_setup_method_fixture_TestTaskManagement",
    "time" : {
      "start" : 1738503493399,
      "stop" : 1738503493399,
      "duration" : 0
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "stepsCount" : 0,
    "hasContent" : false
  }, {
    "name" : "_session_faker",
    "time" : {
      "start" : 1738503462448,
      "stop" : 1738503462449,
      "duration" : 1
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "stepsCount" : 0,
    "hasContent" : false
  } ],
  "testStage" : {
    "status" : "failed",
    "statusMessage" : "AssertionError: Expected status code 400, but got 200. Response: {'status': 'success', 'task': {'_id': '679f75466eb12444f2b46d5e', 'title': 'Test thank subject.', 'start': '07:41:52', 'end': '21:50:11', 'priority': 'someone', 'category': 'in-progress', 'owner': '679f75456eb12444f2b46d5a', 'date': '2025-01-27T00:00:00.000Z', 'createdAt': '2025-02-02T13:38:14.425Z', 'updatedAt': '2025-02-02T13:38:14.910Z', '__v': 0}}",
    "statusTrace" : "self = <test_task_management.TestTaskManagement object at 0x7fcf9fd10f90>\n\n    @pytest.mark.order(33)\n    @allure.title(\"T33: Update an existing task with an invalid priority.\")\n    def test_update_task_with_invalid_priority(self):\n        user = self.api_user_auth.create_user()\n        task = self.api_task.create_task(Token=user.data.accessToken)\n>       self.api_task.update_task_with_invalid_priority(Token=user.data.accessToken, task_id=task.id)\n\ntests/test_task_management.py:83: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \nservices/Task/api_task.py:160: in update_task_with_invalid_priority\n    self.assert_status(response, 400)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <services.Task.api_task.TaskAPI object at 0x7fcf9e41ea50>\nresponse = <Response [200]>, expected_status = 400\n\n    def assert_status(self, response, expected_status):\n>       assert response.status_code == expected_status, \\\n            (f\"Expected status code {expected_status}, but got {response.status_code}. Response: {response.json()}\")\nE       AssertionError: Expected status code 400, but got 200. Response: {'status': 'success', 'task': {'_id': '679f75466eb12444f2b46d5e', 'title': 'Test thank subject.', 'start': '07:41:52', 'end': '21:50:11', 'priority': 'someone', 'category': 'in-progress', 'owner': '679f75456eb12444f2b46d5a', 'date': '2025-01-27T00:00:00.000Z', 'createdAt': '2025-02-02T13:38:14.425Z', 'updatedAt': '2025-02-02T13:38:14.910Z', '__v': 0}}\n\nutils/helper.py:13: AssertionError",
    "steps" : [ {
      "name" : "Create a new user.",
      "time" : {
        "start" : 1738503493400,
        "stop" : 1738503494039,
        "duration" : 639
      },
      "status" : "passed",
      "steps" : [ ],
      "attachments" : [ {
        "uid" : "3b743335b4a477e1",
        "name" : "API Response",
        "source" : "3b743335b4a477e1.json",
        "type" : "application/json",
        "size" : 518
      } ],
      "parameters" : [ ],
      "attachmentsCount" : 1,
      "shouldDisplayMessage" : false,
      "stepsCount" : 0,
      "hasContent" : true
    }, {
      "name" : "Create a new task.",
      "time" : {
        "start" : 1738503494039,
        "stop" : 1738503494522,
        "duration" : 483
      },
      "status" : "passed",
      "steps" : [ ],
      "attachments" : [ {
        "uid" : "1dc6ec624d5995bf",
        "name" : "API Response",
        "source" : "1dc6ec624d5995bf.json",
        "type" : "application/json",
        "size" : 352
      } ],
      "parameters" : [ {
        "name" : "Token",
        "value" : "'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6IjY3OWY3NTQ1NmViMTI0NDRmMmI0NmQ1YSIsImlhdCI6MTczODUwMzQ5MywiZXhwIjoxNzM4NTA3MDkzfQ.ZdqZvQoUr5YpE2CftVjgvSuCI5tj4cdcZpeHbcZCLX4'"
      } ],
      "attachmentsCount" : 1,
      "shouldDisplayMessage" : false,
      "stepsCount" : 0,
      "hasContent" : true
    }, {
      "name" : "Update a task with an invalid priority.",
      "time" : {
        "start" : 1738503494522,
        "stop" : 1738503495010,
        "duration" : 488
      },
      "status" : "failed",
      "statusMessage" : "AssertionError: Expected status code 400, but got 200. Response: {'status': 'success', 'task': {'_id': '679f75466eb12444f2b46d5e', 'title': 'Test thank subject.', 'start': '07:41:52', 'end': '21:50:11', 'priority': 'someone', 'category': 'in-progress', 'owner': '679f75456eb12444f2b46d5a', 'date': '2025-01-27T00:00:00.000Z', 'createdAt': '2025-02-02T13:38:14.425Z', 'updatedAt': '2025-02-02T13:38:14.910Z', '__v': 0}}\n",
      "statusTrace" : "  File \"/usr/local/lib/python3.11/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/usr/workspace/services/Task/api_task.py\", line 160, in update_task_with_invalid_priority\n    self.assert_status(response, 400)\n  File \"/usr/workspace/utils/helper.py\", line 13, in assert_status\n    assert response.status_code == expected_status, \\\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ {
        "name" : "Token",
        "value" : "'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6IjY3OWY3NTQ1NmViMTI0NDRmMmI0NmQ1YSIsImlhdCI6MTczODUwMzQ5MywiZXhwIjoxNzM4NTA3MDkzfQ.ZdqZvQoUr5YpE2CftVjgvSuCI5tj4cdcZpeHbcZCLX4'"
      }, {
        "name" : "task_id",
        "value" : "'679f75466eb12444f2b46d5e'"
      } ],
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : true,
      "stepsCount" : 0,
      "hasContent" : true
    } ],
    "attachments" : [ ],
    "parameters" : [ ],
    "attachmentsCount" : 2,
    "shouldDisplayMessage" : true,
    "stepsCount" : 3,
    "hasContent" : true
  },
  "afterStages" : [ {
    "name" : "_xunit_setup_method_fixture_TestTaskManagement::0",
    "time" : {
      "start" : 1738503495017,
      "stop" : 1738503495017,
      "duration" : 0
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "stepsCount" : 0,
    "hasContent" : false
  } ],
  "labels" : [ {
    "name" : "feature",
    "value" : "Task Management"
  }, {
    "name" : "tag",
    "value" : "order(33)"
  }, {
    "name" : "parentSuite",
    "value" : "tests"
  }, {
    "name" : "suite",
    "value" : "test_task_management"
  }, {
    "name" : "subSuite",
    "value" : "TestTaskManagement"
  }, {
    "name" : "host",
    "value" : "1ae841824f73"
  }, {
    "name" : "thread",
    "value" : "1-MainThread"
  }, {
    "name" : "framework",
    "value" : "pytest"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "package",
    "value" : "tests.test_task_management"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ ],
    "categories" : [ {
      "name" : "Product defects",
      "matchedStatuses" : [ ],
      "flaky" : false
    } ],
    "history" : {
      "statistic" : {
        "failed" : 7,
        "broken" : 0,
        "skipped" : 0,
        "passed" : 0,
        "unknown" : 0,
        "total" : 7
      },
      "items" : [ {
        "uid" : "e87720600780d762",
        "status" : "failed",
        "statusDetails" : "AssertionError: Expected status code 400, but got 200. Response: {'status': 'success', 'task': {'_id': '6793b6bf6eb12444f2b46c78', 'title': 'Shoulder positive.', 'start': '00:30:31', 'end': '06:48:26', 'priority': 'civil', 'category': 'to-do', 'owner': '6793b6bf6eb12444f2b46c74', 'date': '2025-01-14T00:00:00.000Z', 'createdAt': '2025-01-24T15:50:23.844Z', 'updatedAt': '2025-01-24T15:50:24.211Z', '__v': 0}}",
        "time" : {
          "start" : 1737733823074,
          "stop" : 1737733824286,
          "duration" : 1212
        }
      }, {
        "uid" : "155853da0c9ce1be",
        "status" : "failed",
        "statusDetails" : "AssertionError: Expected status code 400, but got 200. Response: {'status': 'success', 'task': {'_id': '678f9ca86eb12444f2b46ba4', 'title': 'Fill parent really.', 'start': '11:52:59', 'end': '07:57:28', 'priority': 'nice', 'category': 'done', 'owner': '678f9ca86eb12444f2b46ba0', 'date': '2025-01-09T00:00:00.000Z', 'createdAt': '2025-01-21T13:10:00.682Z', 'updatedAt': '2025-01-21T13:10:01.037Z', '__v': 0}}",
        "time" : {
          "start" : 1737464999891,
          "stop" : 1737465001116,
          "duration" : 1225
        }
      }, {
        "uid" : "b6c88dca8d732f4d",
        "status" : "failed",
        "statusDetails" : "AssertionError: Expected status code 400, but got 200. Response: {'status': 'success', 'task': {'_id': '678f7fe66eb12444f2b46ad0', 'title': 'In.', 'start': '01:00:54', 'end': '13:17:15', 'priority': 'pressure', 'category': 'to-do', 'owner': '678f7fe66eb12444f2b46acc', 'date': '2025-01-01T00:00:00.000Z', 'createdAt': '2025-01-21T11:07:18.962Z', 'updatedAt': '2025-01-21T11:07:19.365Z', '__v': 0}}",
        "time" : {
          "start" : 1737457638106,
          "stop" : 1737457639447,
          "duration" : 1341
        }
      }, {
        "uid" : "fc3d53f4cb7357ed",
        "status" : "failed",
        "statusDetails" : "AssertionError: Expected status code 400, but got 200. Response: {'status': 'success', 'task': {'_id': '678e57816eb12444f2b469ff', 'title': 'Note sea lay.', 'start': '15:51:25', 'end': '16:38:30', 'priority': 'back', 'category': 'done', 'owner': '678e57806eb12444f2b469fb', 'date': '2025-01-11T00:00:00.000Z', 'createdAt': '2025-01-20T14:02:41.303Z', 'updatedAt': '2025-01-20T14:02:41.663Z', '__v': 0}}",
        "time" : {
          "start" : 1737381760528,
          "stop" : 1737381761741,
          "duration" : 1213
        }
      }, {
        "uid" : "7904dd60d2162674",
        "status" : "failed",
        "statusDetails" : "AssertionError: Expected status code 400, but got 200. Response: {'status': 'success', 'task': {'_id': '678e4cbb6eb12444f2b4692e', 'title': 'Company magazine.', 'start': '01:07:11', 'end': '02:02:17', 'priority': 'energy', 'category': 'done', 'owner': '678e4cba6eb12444f2b4692a', 'date': '2025-01-03T00:00:00.000Z', 'createdAt': '2025-01-20T13:16:43.606Z', 'updatedAt': '2025-01-20T13:16:44.104Z', '__v': 0}}",
        "time" : {
          "start" : 1737379002554,
          "stop" : 1737379004202,
          "duration" : 1648
        }
      }, {
        "uid" : "7d7856b9c9316ba4",
        "status" : "failed",
        "statusDetails" : "AssertionError: Expected status code 400, but got 200. Response: {'status': 'success', 'task': {'_id': '678e49fd6eb12444f2b4685a', 'title': 'Law occur discover.', 'start': '12:14:28', 'end': '20:07:52', 'priority': 'add', 'category': 'done', 'owner': '678e49fc6eb12444f2b46856', 'date': '2025-01-11T00:00:00.000Z', 'createdAt': '2025-01-20T13:05:01.066Z', 'updatedAt': '2025-01-20T13:05:01.246Z', '__v': 0}}",
        "time" : {
          "start" : 1737378300307,
          "stop" : 1737378300972,
          "duration" : 665
        }
      } ]
    },
    "tags" : [ "order(33)" ]
  },
  "source" : "9b4039697c534759.json",
  "parameterValues" : [ ]
}